lib/travis/api/v3/queries/log.rb

```diff
@@ -18,7 +18,7 @@ def find(job)
     end
 
     def create_log_parts(log, content)
-      log.log_parts << Models::LogPart.new(log_id: log.id, content: content, number: 0, created_at: log.created_at)
+      log.log_parts.build([{content: content, number: 0, created_at: log.created_at}])
```

Comment:
ah! ok archived and aggregated logs don't have any log parts in the database so we make one up, cool.